manage/users/index.rst

```diff
@@ -57,7 +60,7 @@ The ones that aren’t are described below.
 .. note:: 🕵️ **Admins aren’t the only ones who can change these settings.**
 
    In most cases, agents can, too
-   (using the `new ticket dialog`_ or the `ticket pane`_).
+   (using the `new ticket dialog`_, search function or the `ticket pane`_).
 
    .. _new ticket dialog: https://user-docs.zammad.org/en/latest/basics/service-ticket/create.html
```

Comment:
```suggestion
   (using the `new ticket dialog`_, `search function`_, or the `ticket pane`_).

   .. _new ticket dialog: https://user-docs.zammad.org/en/latest/basics/service-ticket/create.html
   .. _search bar: https://user-docs.zammad.org/en/latest/basics/find-ticket/search.html
```

Reply:
Good point, much better. 🙈

Reply:
will be address with commit 229e5a8c55fcf709e0eadb232d165c1959a47a63